src/components/parser/linterlog.ts

```diff
@@ -61,11 +65,66 @@ export class LinterLogParser {
         this.showLinterDiagnostics(linterLog)
     }
 
+    private callConvertColumn(column: number, filePathArg: string, line: number, tabSizeArg?: number): number {
+        const configuration = vscode.workspace.getConfiguration('latex-workshop')
+        if (!configuration.get('chktex.convertOutput.column.enabled', true)) {
+            return column
+        }
+        const filePath = convertFilenameEncoding(filePathArg)
+        if (!filePath){
+            this.extension.logger.addLogMessage(`Stop converting chktex's column numbers. File not found: ${filePathArg}`)
+            return column
+        }
+        const lineString = fs.readFileSync(filePath).toString().split('\n')[line-1]
+        let tabSize: number | undefined
+        const tabSizeConfig = configuration.get('chktex.convertOutput.column.chktexrcTabSize', 'auto')
+        if (tabSizeConfig !== 'auto') {
+            tabSize = parseInt(tabSizeConfig)
```

Comment:
Instead of parsing strings here, why not typing `chktex.convertOutput.column.chktexrcTabSize` as a `number` and use `-1` for the auto value for instance? This would avoid the issue with strings not representing integers.

Reply:
Edited as suggested.